deployments/sepolia.json

```diff
@@ -0,0 +1,14 @@
+{
+ "nationToken": "0x23Ca3002706b71a440860E3cf8ff64679A00C9d7",
```

Comment:
@TTNguyenDev Please also _verify_ the contracts:  https://sepolia.etherscan.io/address/0x23Ca3002706b71a440860E3cf8ff64679A00C9d7#code

> ![verify](https://github.com/nation3/foundations/assets/95955389/28ae6b70-f788-402a-a7c0-442a54957997)

Reply:
Okay sure